src/publicApiClient.ts

```diff
@@ -31,6 +32,16 @@ export class PublicApiClient extends AbstractClient {
       .setApiKey(this.apiKey);
   }
 
+  /**
+   * Creates a new {@link LicensesClient} instance and returns it
+   * @returns {@link LicensesClient}
+   */
+  public getLicenseClient(): LicenseClient {
```

Comment:
If you move the method from `LicenseClient` to `LicensesClient` like I suggested, you won't need to add this

Reply:
Done